tests/phpunit/integration/Modules/Reader_Revenue_ManagerTest.php

```diff
@@ -486,6 +486,35 @@ public function test_check_service_entity_access_no_access_unavailable_publicati
 		$this->assertEquals( false, $access );
 	}
 
+	public function test_block_editor_script_enqueued() {
+		$this->enable_feature( 'rrmModuleV2' );
+
+		$registerable_asset_handles = array_map(
+			function ( $asset ) {
+				return $asset->get_handle();
+			},
+			$this->reader_revenue_manager->get_assets()
+		);
+		$this->assertContains(
```

Comment:
Super trivial, let's include a newline here for consistency with the test below :nerd_face: 

I'll apply this myself to save the round trip.

```suggestion

				$this->assertContains(
```